Makefile

```diff
@@ -724,7 +724,7 @@ out/39-ohio/state.gpkg: data/39-ohio/statewide/2010/tl_2012_39_vtd10.zip \
 	ogr2ogr -sql "SELECT '2016' AS year, '39' AS state, '021' AS county, CONCAT('39021', CAST(District_N AS character(10))) AS precinct, 'polygon' AS accuracy FROM CHP_PREC" \
 		-s_srs '+proj=lcc +lat_1=38.73333333333333 +lat_2=40.03333333333333 +lat_0=38 +lon_0=-82.5 +x_0=600000 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=us-ft +no_defs' \
 		-t_srs EPSG:4326 -nln state -append -f GPKG $@ 'out/39-ohio/source/CHP_PREC.shp'
-	unzip -d out/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip
+	unzip -d out/39-ohio/source data/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip
```

Comment:
Hmm, this is a mixed bag:

> unzip -d out/39-ohio/source/39061-hamilton/ data/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip

To match with the later command's path:

> 'out/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.shp'

**or**

> unzip -d out/39-ohio/source/ data/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip

But update the later command's path:

> 'out/39-ohio/HamiltonCountyOHPrecincts_2016.shp'

Reply:
It looks like the common patter for the other files is:

`unzip -d out/39-ohio/source/ data/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip`

and

`'out/39-ohio/source/HamiltonCountyOHPrecincts_2016.shp'`